src/sonic-yang-models/tests/yang_model_tests/tests/mclag.json

```diff
@@ -0,0 +1,36 @@
+{
+    "MCLAG_VALID_DOMAIN_CFG_TEST": {
+	"desc": "valid mclag configuration test"
+    },
+    "MCLAG_VALID_DOMAIN_PEERLINK_ETH_TEST": {
+	"desc": "valid mclag configuration test with peer link ethernet"
+    },
+    "MCLAG_INVALID_DOMAIN_IDS_TEST": {
+	"desc": "configure mclag domain id in range"
+    },
+    "MCLAG_INVALID_KEEPALIVE_TEST": {
```

Comment:
An invalid test must have eStr field, I think. Is this test passing right now.

Reply:
done